redis/command_factory.cc

```diff
@@ -35,6 +35,7 @@ shared_ptr<abstract_command> command_factory::create(service::storage_proxy& pro
         { "ping",  [] (service::storage_proxy& proxy, request&& req) { return commands::ping::prepare(proxy, std::move(req)); } }, 
         { "select",  [] (service::storage_proxy& proxy, request&& req) { return commands::select::prepare(proxy, std::move(req)); } }, 
         { "get",  [] (service::storage_proxy& proxy, request&& req) { return commands::get::prepare(proxy, std::move(req)); } }, 
+        { "exists", [] (service::storage_proxy& proxy, request&& req) { return commands::exists::prepare(proxy, std::move(req)); } },
```

Comment:
Not something for this pull request, but this command lookup table looks pretty verbose, especially since the number of commands is in the hundreds. Perhaps we should simplify it somehow?